store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,66 @@ TODO: delete them permanently from here
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, enables backup globally and if there is pre-configured data the crontab will get that schedule.</desc>
+</attr>
+
+<attr id="4009" name="zimbraBackupSyncUsersEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="9.1.0">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>If true, will synchronize dumpster enabled accounts with backup eligible accounts from zimbraBackupConfigEnabled list.</desc>
+</attr>
+
+<attr id="4010" name="zimbraBackupStartTime" type="string" max="256" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>01:00</globalConfigValue>
+  <desc>Starts backup each day at this time. This attribute is used to set hours and minutes on the crontab.</desc>
+</attr>
+
+<attr id="4011" name="zimbraBackupRetentionDays" type="integer" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>8</globalConfigValue>
+  <desc>The number of days backups will be kept. Backup Scheduler deletes backups older than the set value.</desc>
+</attr>
+
+<attr id="4012" name="zimbraBackupBlobsCompressType" type="string" cardinality="single" max="256" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>zip</globalConfigValue>
+  <desc>Blobs inside a backup are compressed by default in zip format.
+        zip - blobs are backed up into zip files with compression (default).
+        zipStore - blobs are backed up into zip files without compression.
+        noZip - blobs are backed up as individual files rather in zip files.
+  </desc>
+</attr>
+
+<attr id="4013" name="zimbraBackupSyncEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>If true, the full backup runs synchronously.
+        If false, full backup runs in a separate thread and the soap call returns with the backup label.
+  </desc>
+</attr>
+
+<attr id="4014" name="zimbraMailReportEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>TRUE</globalConfigValue>
+  <desc>If true, sends an email report to the admin user. Admins can disable the email notification by setting it to FALSE.</desc>
+</attr>
+
+<attr id="4015" name="zimbraBackupStartDay" type="integer" max="256" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+  <globalConfigValue>7</globalConfigValue>
+  <desc>The day of the week on which the full backup starts.
+        When a day is select their relative value is set on the crontab.
+        0-7 or names allowed, with both 0 and 7 representing Sunday. Monday to Saturday (1-6)
```

Comment:
> names allowed

Then type should not be integer but string, correct?

Reply:
Updated.